src/Handler/EasyHandle.php

```diff
@@ -88,11 +105,9 @@ public function createResponse(): void
     /**
      * @param string $name
      *
-     * @return void
-     *
      * @throws \BadMethodCallException
      */
-    public function __get($name)
+    public function __get($name): void
```

Comment:
do we actually want this?

Reply:
Hm.. this will indeed be problematic for ppl that extend this class.. and it has less benefit indeed. Reverted